pom.xml

```diff
@@ -0,0 +1,149 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.onebusaway</groupId>
+    <artifactId>onebusaway-alexa</artifactId>
+    <version>1.0</version>

```

Comment:
Sorry I missed this the first time - this should be `<version>1.0.0-SNAPSHOT</version>` until we publicly publish an initial release.